app/assets/javascripts/ajax_upload.js

```diff
@@ -3,16 +3,16 @@
 (function($) {
 
   $.fn.ajaxUpload = function() {
-    this.live("submit", function() {
+    $(document).on('submit', this.selector, function () { // FIXME: don't use .selector on jquery objects (jquery 1.9)
```

Comment:
Also, I didn't know about `.live`, but it seems that if `.on` is actually set on the parent which always exists (which is the case, as it's `document`), then the event will still be triggered. It could potentially fail though it it was defined as `$(this.selector).on('submit')`. At least this is from what I understand :grimacing: